INFO: Starting training:
        Epochs:          5
        Batch size:      2
        Learning rate:   2e-05
        Training size:   9001
        Validation size: 1000
        Checkpoints:     True
        Device:          cuda
        Images scaling:  1.0
        Mixed Precision: False
Epoch 1/5:   0%|                                      | 0/9001 [00:00<?, ?img/s]/root/miniconda3/envs/acnet/lib/python3.8/site-packages/torch/nn/functional.py:1960: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
Epoch 1/5:   0%|                                      | 0/9001 [00:03<?, ?img/s]
Traceback (most recent call last):
  File "/root/autodl-tmp/acnet/train.py", line 230, in <module>
    train_net(net=net,
  File "/root/autodl-tmp/acnet/train.py", line 143, in train_net
    loss.backward()
  File "/root/miniconda3/envs/acnet/lib/python3.8/site-packages/torch/_tensor.py", line 396, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph, inputs=inputs)
  File "/root/miniconda3/envs/acnet/lib/python3.8/site-packages/torch/autograd/__init__.py", line 173, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
RuntimeError: cuDNN error: CUDNN_STATUS_INTERNAL_ERROR
/opt/conda/conda-bld/pytorch_1656352465323/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:103: nll_loss2d_forward_kernel: block: [2,0,0], thread: [234,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1656352465323/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:103: nll_loss2d_forward_kernel: block: [2,0,0], thread: [238,0,0] Assertion `t >= 0 && t < n_classes` failed.